webapp-django/crashstats/settings/test.py

```diff
@@ -3,6 +3,7 @@
 """
 
 from crashstats.settings.base import *  # noqa
+from crashstats.settings.base import MIDDLEWARE
```

Comment:
The line above this already imports `MIDDLEWARE`, we can remove this import thanks to that.